### PR TITLE
Regenerate figures with Arial font and generate data. 

### PR DESCRIPTION
**Pull request by iahepburn**
Generate data. Save some large .h5 files and sets as compressed .zip files rothman_plot.py is WIP because it has some unresolved error.

### CODE REVIEW

File: vesicles/validations/vesicle_diff.py [hunk 1]
```diff
@@ -5,12 +5,18 @@
 from steps.rng import *
 from steps.saving import *
 from steps.sim import *
+import steps.simcheck
```
Comment (jlallouette) [inline vesicles/validations/vesicle_diff.py:8]:
```suggestion
```

File: vesicles/validations/vesreac_error.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def run_sim(hdfPath, ntets, D, NITER, use_foi, use_soAA, seed):
         import subprocess
 
         nmpi, totproc, nrunsPerProc, baseseed = map(int, sys.argv[2:])
-        hdfPrefix = 'data/vesreac_error'
+        hdfPrefix = 'data/vesreac_error/vesreac_error'
```
Comment (jlallouette) [inline vesicles/validations/vesreac_error.py:129]: The files do not appear to have been moved on this PR

Comment (iahepburn) [reply vesicles/validations/vesreac_error.py:129]: I've included the .zip file and when you unzip, by default it extracts to` data/vesreac_error/`

Comment (jlallouette) [reply vesicles/validations/vesreac_error.py:129]: I see, so the `data/vesreac_error_xxx.h5` files can be removed then?

Comment (iahepburn) [reply vesicles/validations/vesreac_error.py:129]: We don't have any of those files.

Comment (jlallouette) [reply vesicles/validations/vesreac_error.py:129]: Ah right, I just have them locally. We do have `data/vesreac_immobile_reactants_xxx.h5` that I added at some point, I will zip them and modify the scripts like you did for `vesreac_error`.

Comment (iahepburn) [reply vesicles/validations/vesreac_error.py:129]: We can, but actually they are not as much data as the vesreac_error one so I left them.

**Review by jlallouette — APPROVED** [review]
LGTM!

**Pull request merged**